src/app/story/directives/metrics-downloads.component.ts

```diff
@@ -0,0 +1,164 @@
+          <prx-datepicker [date]="endDate" (dateChange)="endDateChange($event)"></prx-datepicker>
+          <prx-timepicker [date]="endDate" (timeChange)="endDateChange($event)"></prx-timepicker>
+        </span>
+      </p>
+      <p>
+        <label for="interval">Interval:</label>
+        <select id="interval" [ngModel]="interval" (ngModelChange)="intervalChange($event)">
+          <option *ngFor="let i of intervalOptions" [value]="i.value">{{i.name}}</option>
+        </select>
+      </p>
+    </div>
+    <p *ngIf="error" class="error">
+      {{error}}
+    </p>
+    <div *ngIf="!chartData && !error" class="chart-loading"><publish-spinner></publish-spinner></div>
+    <prx-line-timeseries-chart *ngIf="chartData" [datasets]="chartData" [dateFormat]="dateFormat"></prx-line-timeseries-chart>
+  `,
+  styleUrls: ['metrics-downloads.component.css']
+})
+
+export class MetricsDownloadsComponent {
+  INTERVAL_DAILY = {value: '1d', name: 'daily'};
+  INTERVAL_HOURLY = {value: '1h', name: 'hourly'};
+  INTERVAL_15MIN = {value: '15m', name: '15 minutes'};
+  intervalOptions = [this.INTERVAL_15MIN, this.INTERVAL_HOURLY, this.INTERVAL_DAILY];
+  story: StoryModel;
+  episode: FeederEpisodeModel;
+  tabSub: Subscription;
+  dateFormat = '%m/%d';
+  beginDate: Date = new Date(moment().subtract(7, 'days').valueOf());
+  endDate: Date = new Date(moment().add(1, 'days').valueOf());
+  interval = this.INTERVAL_DAILY.value;
+  error: string;
+  chartData: TimeseriesChartModel[];
+
+  constructor(tab: TabService, private castle: CastleService) {
+    this.beginDate.setHours(0, 0, 0);
+    this.endDate.setHours(0, 0, 0);
+    this.tabSub = tab.model.subscribe((s: StoryModel) => {
+      this.story = s;
+      this.story.loadRelated('distributions', true).subscribe(() => {
+        let storyDistribution = this.story.distributions.find(d => d.kind === 'episode');
+        if (storyDistribution) {
+          storyDistribution.loadRelated('episode').subscribe(() => {
+            this.episode = storyDistribution.episode;
+            this.requestMetrics();
+          });
+        }
+      });
+    });
+  }
+
+  requestMetrics() {
+    this.chartData = null;
+    if (this.checkRequest()) {
+      this.castle.followList('prx:episode-downloads', {
+        guid: this.episode.id,
+        from: moment(this.beginDate).format(),
+        to: moment(this.endDate).format(),
+        interval: this.interval
+      }).subscribe(metrics => {
+        if (metrics[0] && metrics[0]['downloads'] && metrics[0]['downloads'].length > 0) {
+          let dataset = metrics[0]['downloads'].map(datum => {
+            return new TimeseriesDatumModel(datum[1], moment(datum[0]).valueOf());
+          });
+          this.chartData = [new TimeseriesChartModel(dataset, this.story.title, '#61A85D')];
+        } else {
+          this.error = 'This podcast has no download metrics.';
+        }
+      });
+    }
+  }
+
+  checkRequest() {
+    if (!this.story || !this.story.publishedAt) {
```

Comment:
Oh, okay, so it's actually that my series has no podcast-distribution.

I'm just going to do a quick fix to hide the tab for new-or-podcast-less stories.  And throw up an error message if Castle returns a non-200.